src/details/ArborX_DetailsCartesianGrid.hpp

```diff
@@ -17,10 +17,13 @@
 #include <ArborX_HyperBox.hpp>
 
 #include <Kokkos_Macros.hpp>
+#if KOKKOS_VERSION >= 40199
```

Comment:
kokkos/kokkos#6445 was merged relatively late after we branched and release 4.1
```suggestion
#if KOKKOS_VERSION >= 40200
```
It is safer to guard against 4.2

Reply:
Agree with that.